spec/requests/api/v0/post_tokens_spec.rb

```diff
@@ -2,35 +2,27 @@
 
 describe 'POST /api/v0/tokens' do
   it 'returns a 401 and no token when auth fails' do
-    VCR.use_cassette('auth_failure') do
```

Comment:
after spending some time with this, my suggestion would be to use the VCR approach with the unit tests in `spec/models/cdx` and the mock approach with everything else. That gives the most granular control and reduces the maintenance burden on us to keep VCR scripts in sync with reality.